website/pages/docs/internals/acl.mdx

```diff
@@ -13,5 +13,6 @@ description: >-
 This content has been moved into the [ACL Guide](https://learn.hashicorp.com/consul/security-networking/production-acls).
 
 <a name="version_8_acls"></a>
```

Comment:
This line probably **shouldn't** be necessary, in which we can just delete this anchor link